### PR TITLE
[6] Product Page using HTML

### PR DESCRIPTION
**Pull request by Benedictcares**
## Task: `create a product page using only HTML
Capture 1
Capture 2

`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/ejikeIweriebor/html/product_page.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/ejikeIweriebor/html/product_page.html:1]: Well done!

File: submissions/ejikeIweriebor/html/product_page.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<META CHARSET="UTF-8">
```
Comment (ToluAkin) [inline submissions/ejikeIweriebor/html/product_page.html:4]: `meta` attribute should be in lowercase.

File: submissions/ejikeIweriebor/html/product_page.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+        <nav>
+            <a href="#">Men</a>
+            <a href="#">Women</a>
+            <a href="#">Kiddies</a>
+            <a href="#">Electronics</a>
+            <a href="#">Home</a>
+        </nav>
+
+        <section>
+            <img src="vans1.png" alt="vans" width="80" Height="80">
```
Comment (ToluAkin) [inline submissions/ejikeIweriebor/html/product_page.html:21]: Remove the width and height from the images.

File: submissions/ejikeIweriebor/html/product_page.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+            </div>
+        
+        </section>
+
+        <section>
+
+            <h2>Similar Products</h2>
+            <ol>
+               
+                  <li><img src="vans-alternative.png" alt="shoe alternative" height=60 width=60><p><a href="#">MultiColoured Vans Shoes</a></p>
```
Comment (ToluAkin) [inline submissions/ejikeIweriebor/html/product_page.html:62]: You can ONLY use 2 product images.

**Comment by Benedictcares** [comment]
It has been updated

**Pull request merged**